README.md

```diff
@@ -6,33 +6,103 @@ This plugin allows you to mount remote folder using sshfs in your container easi
 
 ## Usage
 
+### Use with SSH keys
 1 - Install the plugin
 
 ```
 $ docker plugin install vieux/sshfs # or docker plugin install vieux/sshfs DEBUG=1
+Plugin "vieux/sshfs" is requesting the following privileges:
+ - network: [host]
+ - mount: [/tmp]
+ - device: [/dev/fuse]
+ - capabilities: [CAP_SYS_ADMIN]
+Do you grant the above permissions? [y/N] y
+latest: Pulling from vieux/sshfs
+e248a6530152: Download complete
+Status: Downloaded newer image for vieux/sshfs:latest
+Installed plugin vieux/sshfs
+```
+
+2 - Configure driver to point to your SSH keys
+
+_NOTE_: The plugin defaults to looking for SSH keys in `/tmp`. You can copy your `<identity_file_name>` to `/tmp` and omit this step, e.g.
```

Comment:
Ying is correct in that environment variables like $HOME can not be used in the plugin's `config.json` file (which defines the privileges the plugin will request.

Some things to bear in mind about how plugins work:
1. The `mount` path defined by the plugin _must_ exist on the user's machine prior to performing the plugin install, or else the install will fail.

2. Installing a plugin causes the plugin to not only be downloaded, but also _enabled_. This means that granting the requested privileges during the install is the last chance anyone has to limit the scope of what the plugin can do before the plugin's code begins executing. So whatever path we set as the default path is readable by the plugin as soon as it is installed. 

I chose `/tmp` as a compromise between "must exist on a wide collection of OS families" and "least likely to contain sensitive user or system files at the time the plugin is installed"